yt_dlp/extractor/franceculture.py

```diff
@@ -44,18 +54,42 @@ def _real_extract(self, url):
             ''',
             webpage, 'video data'))
 
-        video_url = video_data.get('data-url') or video_data['data-asset-source']
-        title = video_data.get('data-asset-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
-
+        title = video_data.get('data-emission-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
```

Comment:
u can combine the extraction from video_data by using traverse_obj

Reply:
Tried `title = traverse_obj(video_data, ('data-emission-title', 'data-diffusion-title'), self._og_search_title(webpage))` but it returns None

Reply:
```suggestion
        title = title = traverse_obj(video_data, 'data-emission-title', 'data-diffusion-title', default=self._og_search_title(webpage))
```
iirc the code you entered would've done `video_data.get('data-emission-title', {}).get''data-diffusion-title'') or video_data.get(self._og_search_title(webpage))`

Reply:
Great, done thanks :)